spec.html

```diff
@@ -20788,7 +20788,7 @@ <h1>
           Runtime Semantics: RestDestructuringAssignmentEvaluation (
             _value_: unknown,
             _excludedNames_: unknown,
-          ): either a normal completion containing ~unused~ or an abrupt completion
+          ): either a normal completion containing ~unused~ or a throw completion
```

Comment:
This can produce a return completion, as in

```js
(function* (){ ({ ...{}[yield 42] } = {}) })().return(42)
```

Reply:
Fixed. (I was avoiding "Evaluation" ops on principle, regardless of what my analysis said, but that one slipped through.)